### ISSUE TITLE
FileUpload.js TypeError: Cannot set property 'value' of undefined

### ISSUE THREAD

**Issue by reefki** [livewire/livewire#2091]
### Description
Every time I click the file input I got a JS error:

```
Uncaught TypeError: Cannot set property 'value' of undefined
    at HTMLInputElement.clearFileInputValue (FileUploads.js:37)
```

### Stripped-down, copy-pastable code snippets
```
// Component

<?php

namespace App\Http\Livewire\User;

use Livewire\Component;
use Livewire\WithFileUploads;

class UserAvatar extends Component
{
    use WithFileUploads;

    public $avatar;

    public function rules()
    {
        return [
            'avatar' => [
                'image',
            ]
        ];
    }

    public function render()
    {
        return view('livewire.user.user-avatar');
    }
}
```

```
// View

<div>
    <input type="file" wire:model="avatar">
</div>
```

**Context**
- Livewire version: 2.3.2
- Laravel version: 8.15.0
- Alpine version: 2.7.3
- Browser: Chrome

**Comment by mbardelmeijer** [comment]
I'm having the same issue while implementing the spatie Laravel Medialibrary Pro library.

```
<!DOCTYPE html>
<html>
    <head>
        <meta charset="utf-8">
        <meta name="viewport" content="width=device-width, initial-scale=1">

        <title>Laravel</title>

        <link href="{{ mix('css/app.css') }}" rel="stylesheet">

        <livewire:styles />
        <livewire:scripts />

        <script src="https://cdn.jsdelivr.net/gh/alpinejs/alpine@v2.6.0/dist/alpine.min.js" defer></script>

    </head>
    <body class="antialiased">
        <div class="relative flex items-top justify-center min-h-screen bg-gray-100 dark:bg-gray-900 sm:items-center sm:pt-0">
            <form method="POST">
                @csrf

                <x-media-library-attachment name="avatar" />

                <button type="submit">Submit</button>
            </form>
        </div>
    </body>
</html>
```

**Comment by mbardelmeijer** [comment]
Ref. https://github.com/livewire/livewire/pull/2055

**Comment by squishythejellyfish** [comment]
👋 Oh Hi! I'm Squishy, the friendly jellyfish that manages Livewire issues.

I see this issue has been closed.

Here in the Livewire repo, we have an "issues can be closed guilt-free and without explanation" policy.

If for ANY reason you think this issue hasn't been resolved, PLEASE feel empowered to re-open it.

Re-opening actually helps us track which issues are a priority.

Reply "REOPEN" to this comment and we'll happily re-open it for you!

(More info on this philosophy here: https://twitter.com/calebporzio/status/1321864801295978497)

**Comment by calebporzio** [comment]
fixed in release v2.3.4

**Comment by reefki** [comment]
@calebporzio fantastic! Thank you
